CONTRIBUTING.md

```diff
@@ -17,11 +17,20 @@ We have two types of documentation:
 - Markdown files, which can be found in the [docs](docs/website/contents)
   directory. They contain information that is not strictly related to the code
   itself, such as getting started guides, references, tutorials, etc.
+
+  They are rendered at the [Consensus
+  website](https://ouroboros-consensus.cardano.intersectmbo.org).
 - [Haddock][haddock-site] comments. They contain more low level information
   about the code and should be used as a reference, although we aim to provide
   some context navigation (cross referencing modules) on the module
   descriptions.
 
+When starting to work on Consensus, we recommend to take a look at the following
+resources:
+
+ - [Preflight guide](https://ouroboros-consensus.cardano.intersectmbo.org/docs/for-developers/PreflightGuide)
```

Comment:
Page not found

Reply:
This link will be valid once this PR has been merged to `main` as this is exactly the page that is being added in this PR; we don't deploy our site from PRs.